packages/angular/package.json

```diff
@@ -10,30 +10,30 @@
   },
```

Comment:
And in this file, I thinks we should update the package `name` field to be something like `@oneschema/angular-example`.

Reply:
Ideally, I think we would have these two projects (the library and the example) be in separate folders without any overlaps. I'm not sure how happy `ng` tools would be, though.